javascript-arrays/main.js

```diff
@@ -0,0 +1,27 @@
+colors =  ['red', 'white', 'blue'];
```

Comment:
you should be declaring the variable here, otherwise you are creating an autoglobal and that is something you really want to avoid. Make sure to fix this and the one below for students.